problemsets-mat223/modules/module2-exercises.tex

```diff
@@ -53,7 +80,26 @@
 				parallel to $\mathcal P_2$, but which does not
 				intersect $\ell$? Why or why not?
 		\end{enumerate}
-
+        % Q4 Solutions
+        \begin{solution}
+            \begin{enumerate}
+                \item $\mathcal P_1\cap \ell = \Set*{\mat{1\\1\\0}}$
+                \item $\mathcal P_1\cap \mathcal P_2 = \Set*{s\mat{1\\1\\3} + \mat{\frac{-1}{2}\\ \frac{3}{2}\\ \frac{-1}{2}} : s \in \R}$
+                \item $\mathcal P_2\cap \ell = \Set*{\mat{2\\4\\1}}$
+                \item $\vec v = s\mat{0\\0\\1} + t\mat{1\\3\\1}$ (Answers may vary)
+                \item No. Since $P_2\cap \ell$ has one vector, the row reduction of $\mat{1 & 0 & 1 \\ 3 & 0 & 1 \\ 1 & 2 & 1 }$ has three pivots. If $P_2'$ is parallel to $P_2$, then $P_2'\cap \ell$ consists of vectors whose coordinates satisfy a system of linear equations corresponding to an augmented matrix of the form 
+                \[
+                \left[
+                    \begin{array}{ccc|c}
+                    1 & 0 & 1  & ? \\
+                    3 & 0 & 1  & ? \\
+                    1 & 2 & 1  & ? \\
+                    \end{array}
+                \right]
+                \]
+                , which is consistent since the now reduced form has three pivots.
```

Comment:
A comma at the start of a line?

I think this explanation can be more clearly worded.